nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseRecord.java

```diff
@@ -317,18 +333,34 @@ protected PutFlowFile createPut(ProcessContext context, Record record, RecordSch
 
         final byte[] fam  = clientService.toBytes(columnFamily);
 
+        final Long timestamp;
+        if (!StringUtils.isBlank(timestampFieldName)) {
+            try {
+                timestamp = record.getAsLong(timestampFieldName);
```

Comment:
Could the record be null here? (I assume it could since we're checking the record is not null few lines below)